tests/test_hindcasting.py

```diff
@@ -80,14 +82,25 @@ def test_hindcasting_GEPS(self, get_local_testdata, salmon_hru, tmp_path):
 
         out = EnsembleReader(runs=ens)
 
-        # The model now has the forecast data generated and it has 5 days of forecasts.
+        # The model now has the forecast data generated, and it has 5 days of forecasts.
         assert len(out.hydrograph.member) == 3
         assert len(out.hydrograph.time) == 5
 
+    # Skip if using Python3.10
+    @pytest.mark.skipif(
+        (3, 11) > sys.version_info >= (3, 10),
+        reason="climpred is unstable in Python 3.10",
+    )
     def test_climpred_hindcast_verif(self, get_local_testdata, salmon_hru, tmp_path):
         ts = get_local_testdata(
             "raven-gr4j-cemaneige/Salmon-River-Near-Prince-George_meteo_daily.nc"
         )
+        # Make a local copy to evade double-ownership of file - first file
+        ts_tmp1 = tmp_path / "salmon_river_near_prince_george-tmp1.nc"
+        ts_tmp1.write_bytes(ts.read_bytes())
+        # Make a local copy to evade double-ownership of file - second file
+        ts_tmp2 = tmp_path / "salmon_river_near_prince_george-tmp2.nc"
+        ts_tmp2.write_bytes(ts.read_bytes())
```

Comment:
Do you know why is this causing problems ? Should that copying logic be embedded in `get_local_testdata` directly ?

Reply:
I added this in as an extra check, but I'm beginning to doubt whether this does anything. The issue appears to be that the file is modified in place and then re-opened later on _within the same function call__. Copying files passed to the function has no impact.

Reply:
I'll leave it for now, as we still need to overhaul the system with `pooch`. I'll be removing lots of code like this then.